docker-compose.yml

```diff
@@ -1,19 +1,21 @@
 version: '2'
 services:
   web:
-    image: fpfis/httpd-php-dev:7.3
+    image: fpfis/httpd-php-dev:8.0
```

Comment:
Why are we using the `fpfis/httpd-php` image? Task Runner is only dealing with PHP on CLI level, it doesn't need a webserver (Apache, Nginx, etc). Using an image such as `php:8.0-cli` would allow us to be more generic and not coupled to EC. Remember, Task Runner is supposed to be an opensource tool.

Also, this would allow to test also with PHP 8.1 as FPFIS image for 8.1 is not yet out.